make.py

```diff
@@ -45,7 +54,7 @@
             coords.append((float(node.lon), float(node.lat)))
         collection.append(LineString(coords))
     shape = await make_polygon(conn, MultiLineString(collection))
-    return shape, relation.tags
+    return shape, {}
```

Comment:
Why returning a tuple in the end now? Seems like we could return only the `shape`, no?

Reply:
Should be ok now : https://github.com/tilery/mae-boundaries/pull/7/commits/f14bdc95bb3eb583f3cfd64e44f896d424940a51